src/shared/lib/oql/AccessorsForOqlFilter.ts

```diff
@@ -149,12 +149,25 @@ export default class AccessorsForOqlFilter
         );
     }
 
+    // public gene(d: Datum) {
+    //     if (
+    //         this.molecularAlterationType(d.molecularProfileId) ===
+    //         AlterationTypeConstants.STRUCTURAL_VARIANT
+    //     ) {
+    //         return (d as StructuralVariant).site1HugoSymbol;
+    //     }
+    //     return (d as any).gene.hugoGeneSymbol;
+    // }
+
     public gene(d: Datum) {
         if (
             this.molecularAlterationType(d.molecularProfileId) ===
             AlterationTypeConstants.STRUCTURAL_VARIANT
         ) {
-            return (d as StructuralVariant).site1HugoSymbol;
+            return [
```

Comment:
@adamabeshouse it's odd that TS allows me to return an array here, no?  I would expect that IAccessorsForOqlFilter interface to enforce string return type.

Reply:
yeah, I have no idea why that is.